cmd/acikubectl/cmd/debug.go

```diff
@@ -219,6 +217,20 @@ func clusterReport(cmd *cobra.Command, args []string) {
 	}
 }
 
+
+func getOutfile(output string) (string, *os.File, error)  {
+	if output == "-" {
+		return "standard output", os.Stdout, nil
+	} else {
+	        outfile, err := os.Create(output)
+		if err != nil {
+		        fmt.Fprintln(os.Stderr, err)
```

Comment:
don't output the error here; if you're bubbling it up output the error in the caller.  i.e. just return the error returned by os.Create